src/Sulu/Bundle/AdminBundle/Resources/js/views/FormOverlayList/ErrorSnackbar.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import {translate} from '../../utils/Translator';
+import Icon from '../../components/Icon';
+import errorSnackbarStyles from './errorSnackbar.scss';
+
+type Props = {|
+    onCloseClick?: () => void,
+    visible: boolean,
+|};
+
+const ICON = 'su-exclamation-triangle';
+
+export default class ErrorSnackbar extends React.Component<Props> {
```

Comment:
Isn't this class almost identical to the `Snackbar` component in the toolbar? Maybe we can extract and reuse it? What's the reason for not doing that?

Reply:
i actually copied that class and removed everything but the error case 😉 .
i am open to extracting et into a separate component - where would we store that one?